src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -15,7 +15,11 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     public static partial class SymbolFinder
     {
         /// <summary>
-        /// Finds the symbol that is associated with a position in the text of a document.
+        /// Finds the symbol that is associated with a position in the text of a document. If 
+        /// multiple symbols are associated with the token (e.g. implicitly named anonymous type
```

Comment:
Invoking @jasonmalinowski to chime in again, because I think he expressed a desire to keep this returning one symbol (he probably had a more nuanced stance than that). We have other heuristics at play here too -- if the position is touching two tokens, we pick only one to bind. If we just make ```IEnumerable<ISymbol> FindSymbolsAtPosition(...)```, we won't be returning them enough info to distinguish between defined & referenced symbols, or to figure out which symbols came from which token (if the position touches two tokens), so they'd end up having to call those individual APIs anyway.